build_tools/docker/README.md

```diff
@@ -110,3 +133,5 @@ modify the `docker` images.
     ```shell
     python3 build_tools/docker/manage_prod.py
     ```
+
+[gcloud-help]: https://cloud.google.com/container-registry/docs/advanced-authentication#gcloud-helper
```

Comment:
I much prefer inline links unless they're *super* complex and/or only useful in the rendered markdown. Markdown should generally be readable and usable in either the source or rendered html format and if the links get stuck at the bottom, a reader of the source has to go hunting around for them.

```suggestion
```